amundsen_application/static/css/_variables-default.scss

```diff
@@ -62,6 +62,19 @@ $badge-primary-color: $cyan10;
 $badge-success-color: $mint20;
 $badge-warning-color: $amber30;
 
+$badge-negative-color: $sunset20;
+$badge-neutral-color: $gray20;
+$badge-positive-color: $mint20;
+$badge-warning-color: $amber30;
+
+$badge-overlay: $gray100;
+$opacity-negative-neutral-positive-primary: .14;
+$opacity-warning: .16;
+$opacity-pressed-negative-neutral-positive-primary: .21;
+$opacity-pressed-warning: .22;
```

Comment:
I would suggest something like:
$badge-opacity-light
$badge-opacity-medium
$badge-opacity-dark

I would try to get those 0.21 and 0.22 into the same one

Reply:
yeah I think that naming is way more scalable ahahaha